html_parser/ga_html_parser.py

```diff
@@ -927,12 +955,14 @@ def write_soup_to_file(self):
             - write html str to an output file
         """
         soup_str = str(self.soup.prettify(formatter=None))
+
+
         for tag in self.meta_tags:
             cleansed_tag = re.sub(r'/>', ' />', str(tag))
             soup_str = re.sub(rf'{tag}', rf'{cleansed_tag}', soup_str, re.I)
         print("validating")
         # html5validate.validate(soup_str)
-        with open(f"../transforms/ga/ocga/r{self.release_number}/{self.html_file_name}", "w") as file:
+        with open(f"/home/mis/cic-code-ga/transforms/ga/ocga/r{self.release_number}/{self.html_file_name}", "w") as file:
```

Comment:
@yogesh-UC hardcoded path

Reply:
@glevita-uc can you fix this

Reply:
@glen-uc @yogesh-UC  this issue is fixed